packaging/conda/meta.yaml

```diff
@@ -52,18 +52,27 @@ outputs:
         - {{ mpi }}
     files:
       - bin/MODL*
+      - bin/_khiopsgetprocnumber*
+      - bin/khiops_env # [linux or osx]
+      - bin/khiops # [linux or osx]
+      - bin/khiops_coclustering # [linux or osx]
+      - bin/khiops_env.cmd # [win]
+      - bin/khiops.cmd # [win]
+      - bin/khiops_coclustering.cmd # [win]
     test:
       commands:
-        - MODL -s
-        - MODL_Coclustering -s
+        - khiops_env --env 
+        - khiops -s
+        - khiops_coclustering -s
```

Comment:
I got the point of using `khiops` and `khiops_coclustering` for tests _in the CI_. But for the Conda package, IMHO it would be better if we at least also tested `MODL` and `MODL_Coclustering` directly, as it would still be those that would be consumed by the Python library.